pkg/image/api/v1/types.go

```diff
@@ -45,50 +45,50 @@ type ImageStream struct {
 	kapi.ObjectMeta `json:"metadata,omitempty"`
 
 	// Spec describes the desired state of this stream
-	Spec ImageStreamSpec `json:"spec"`
+	Spec ImageStreamSpec `json:"spec" description:"desired state of the stream"`
 	// Status describes the current state of this stream
-	Status ImageStreamStatus `json:"status,omitempty"`
+	Status ImageStreamStatus `json:"status,omitempty" description:"current state of the stream as observed by the system"`
 }
 
 // ImageStreamSpec represents options for ImageStreams.
 type ImageStreamSpec struct {
 	// Optional, if specified this stream is backed by a Docker repository on this server
-	DockerImageRepository string `json:"dockerImageRepository,omitempty"`
+	DockerImageRepository string `json:"dockerImageRepository,omitempty" description:"optional field if specified this stream is backed by a Docker repository on this server"`
 	// Tags map arbitrary string values to specific image locators
-	Tags []NamedTagReference `json:"tags,omitempty"`
+	Tags []NamedTagReference `json:"tags,omitempty" description:"map arbitrary string values to specific image locators"`
 }
 
 // NamedTagReference allows a user to TODO.
```

Comment:
Since I accidentally left this TODO, could you make this

NamedTagReference specifies optional annotations for images using this tag and an optional reference to another ImageStreamTag or ImageStreamImage this tag should track.